javascript/roman-numeral-converter.js

```diff
@@ -0,0 +1,87 @@
+(function () {
+  function converter(rootE) {
+    const form = rootE.querySelector(".converter__form");
+    const convertButton = rootE.querySelector(".converter__form__fieldset-btn");
+    const output = rootE.querySelector(".converter__output");
+    const input = rootE.querySelector(".converter__form__fieldset-input")
+
+    const convertToRoman = (num) => {
+      const ref = [
+        ["M", 1000],
+        ["CM", 900],
+        ["D", 500],
+        ["CD", 400],
+        ["C", 100],
+        ["XC", 90],
+        ["L", 50],
+        ["XL", 40],
+        ["X", 10],
+        ["IX", 9],
+        ["V", 5],
+        ["IV", 4],
+        ["I", 1],
+      ];
+      const res = [];
+
+      ref.forEach(function (arr) {
+        while (num >= arr[1]) {
+          res.push(arr[0]);
+          num -= arr[1];
+        }
+      });
+
+      return res.join("");
+    };
+
+    const isValid = (str, int) => {
+      let errText = "";
+
+      if (!str || str.match(/[e.]/g)) {
+        errText = "Please enter a valid number.";
+      } else if (int < 1) {
+        errText = "Please enter a number greater than or equal to 1.";
+      } else if (int > 3999) {
+        errText = "Please enter a number less than or equal to 3999.";
```

Comment:
Try putting this text in different HTML tags in the HTML template and hide them with css.
Then this logic should find the appropriate element and display it correctly.
Always try as much to put everything related to content in the HTML as it's not a good practice to put content directly in the JS code